src/components/spinner/index.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { FC } from 'react';
+import { Spinner as ChakraSpinner } from '@chakra-ui/react';
+
+type Props = {
+  size: 'xs' | 'sm' | 'md' | 'lg';
```

Comment:
Would it make sense to read that from the size tokens (and exclude the ones we want to exclude)?

Reply:
It feels like a bit overkill to me and we would need to align other components like the button with that approach.
If you think it's worth the effort feel free to creat a task for it.